app/src/main/java/com/sopt/kakaobank/presentation/home/HomeItem.kt

```diff
@@ -0,0 +1,8 @@
+package com.sopt.kakaobank.presentation.home
+
+sealed class HomeItem {
+    data class CheckLimitItem(val username: String) : HomeItem()
+    data class BankBookItem1(val name: String, val leftover: String) : HomeItem()
+    data class BankBookItem2(val name: String, val leftover: String) : HomeItem()
+    data class BankBookItem3(val name: String, val leftover: String, val withdraw: String) : HomeItem()
+}
```

Comment:
```suggestion
sealed interface HomeItem {
    data class CheckLimitItem(val username: String) : HomeItem
    data class BankBookItem1(val name: String, val leftover: String) : HomeItem
    data class BankBookItem2(val name: String, val leftover: String) : HomeItem
    data class BankBookItem3(val name: String, val leftover: String, val withdraw: String) : HomeItem
}
```
두 코드의 차이점을 서술하시오!(고치라는건 아닙니당)